From 2b5ea4b76f490dd33702a90ab9d650a0fbcf9392 Mon Sep 17 00:00:00 2001
From: Liu Ying <victor.liu@nxp.com>
Date: Tue, 8 May 2018 11:08:27 +0800
Subject: [PATCH 3750/5242] MLK-18229-4 gpu: imx: dpu: fetchwarp: Cosmetic
 changes on fw_ops entries

commit  02c1772cc9f107a76f55405aaac9f207e13e2d63 from
https://source.codeaurora.org/external/imx/linux-imx.git

This patch contains cosmetic changes on fw_ops entries to make the
entries be more readable.

Signed-off-by: Liu Ying <victor.liu@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/gpu/imx/dpu/dpu-fetchwarp.c |   32 ++++++++++++++++----------------
 1 file changed, 16 insertions(+), 16 deletions(-)

diff --git a/drivers/gpu/imx/dpu/dpu-fetchwarp.c b/drivers/gpu/imx/dpu/dpu-fetchwarp.c
index 99ef232..673c738 100644
--- a/drivers/gpu/imx/dpu/dpu-fetchwarp.c
+++ b/drivers/gpu/imx/dpu/dpu-fetchwarp.c
@@ -241,22 +241,22 @@ void dpu_fw_put(struct dpu_fetchunit *fu)
 EXPORT_SYMBOL_GPL(dpu_fw_put);
 
 static const struct dpu_fetchunit_ops fw_ops = {
-	.set_burstlength = fetchunit_set_burstlength,
-	.set_baseaddress = fetchunit_set_baseaddress,
-	.set_src_bpp = fetchunit_set_src_bpp,
-	.set_src_stride = fetchunit_set_src_stride,
-	.set_src_buf_dimensions = fetchwarp_set_src_buf_dimensions,
-	.set_fmt = fetchwarp_set_fmt,
-	.enable_src_buf = fetchunit_enable_src_buf,
-	.disable_src_buf = fetchunit_disable_src_buf,
-	.is_enabled = fetchunit_is_enabled,
-	.set_framedimensions = fetchwarp_set_framedimensions,
-	.set_controltrigger = fetchwarp_set_controltrigger,
-	.get_stream_id = fetchunit_get_stream_id,
-	.set_stream_id = fetchunit_set_stream_id,
-	.pin_off = fetchunit_pin_off,
-	.unpin_off = fetchunit_unpin_off,
-	.is_pinned_off = fetchunit_is_pinned_off,
+	.set_burstlength	= fetchunit_set_burstlength,
+	.set_baseaddress	= fetchunit_set_baseaddress,
+	.set_src_bpp		= fetchunit_set_src_bpp,
+	.set_src_stride		= fetchunit_set_src_stride,
+	.set_src_buf_dimensions	= fetchwarp_set_src_buf_dimensions,
+	.set_fmt		= fetchwarp_set_fmt,
+	.enable_src_buf		= fetchunit_enable_src_buf,
+	.disable_src_buf	= fetchunit_disable_src_buf,
+	.is_enabled		= fetchunit_is_enabled,
+	.set_framedimensions	= fetchwarp_set_framedimensions,
+	.set_controltrigger	= fetchwarp_set_controltrigger,
+	.get_stream_id		= fetchunit_get_stream_id,
+	.set_stream_id		= fetchunit_set_stream_id,
+	.pin_off		= fetchunit_pin_off,
+	.unpin_off		= fetchunit_unpin_off,
+	.is_pinned_off		= fetchunit_is_pinned_off,
 };
 
 void _dpu_fw_init(struct dpu_soc *dpu, unsigned int id)
-- 
1.7.9.5

